.eslintrc.js

```diff
@@ -8,7 +15,7 @@ module.exports = {
     "SharedArrayBuffer": "readonly"
   },
   "rules": {
-    "no-unused-vars": "error",
-    "no-var": "error"
+    "no-var": "error",
+    "@typescript-eslint/no-unused-vars": ["error"]
```

Comment:
I think we will also need to preserve the original `no-unused-vars` rule for JS code.

Reply:
This rule also works with `js` files.

If add origin `no-unused-vars` back, will get two similar errors in `js` files.

```
  2:17  error  'validators' is assigned a value but never used  no-unused-vars
  2:17  error  'validators' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

and in `ts` files

```
  16:29  error  'Set' is defined but never used  no-unused-vars
  16:29  error  'Set' is defined but never used  @typescript-eslint/no-unused-vars
```

And if I only use original `no-unused-vars`, some imported variables only used for types will cause this error.

```
  1:10  error  'Address' is defined but never used  no-unused-vars
  1:19  error  'Script' is defined but never used   no-unused-vars
```

Reply:
Gotcha.